lib/roger/info.ex

```diff
@@ -67,20 +67,21 @@ defmodule Roger.Info do
   defp get_queue_messages(_, _, 0, result) do
     result
   end
+
   defp get_queue_messages(channel, queue, count, acc) do
     case AMQP.Basic.get(channel, queue, no_ack: false) do
       {:ok, payload, _meta} ->
         {:ok, job} = Job.decode(payload)
         get_queue_messages(channel, queue, count - 1, [job | acc])
+
       {:empty, _} ->
         acc
     end
   end
 
-
   defp gather(call, args \\ []) do
-    {:ok, result} = System.call({:apply, Roger.NodeInfo, call}, args)
-    result
+    [:this, :visible]
```

Comment:
System.call determines the nodes via line: `nodes = [node() | :erlang.nodes()]`. Is this new way of fetching the node 100% compatible?